### PR TITLE
Tagset/Reference components

### PR DESCRIPTION
**Pull request by atstojanov**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated
- [ ] I've read the contributing document https://github.com/cherrytwist/.github/blob/master/CONTRIBUTING.md
- [ ] I've read and understand the Code of Conduct https://github.com/cherrytwist/.github/blob/master/CODE_OF_CONDUCT.md
- [ ] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/cherrytwist/Coordination/blob/master/LICENSE 
     and the contributor license agreement: tba

### CODE REVIEW

File: src/components/Admin/Common/InputField.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <Form.Control
+        name={name}
+        as={as ? as : 'input'}
+        type={type || 'text'}
+        placeholder={placeholder || title}
+        value={value}
+        onChange={field.onChange}
+        required={required}
+        readOnly={readOnly}
+        disabled={readOnly}
```
Comment (valentinyanakiev) [inline src/components/Admin/Common/InputField.tsx:43]: disabled should be removed

File: src/models/Profile.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export type Reference = {
```
Comment (valentinyanakiev) [inline src/models/Profile.ts:1]: Move all profile entities to Profile.ts and use them from here (at the moment this type is not referenced anywhere and overlaps with the one in User.ts)

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]

File: src/components/Admin/Common/InputField.tsx (hunk 1 above)
Comment (atstojanov) [reply src/components/Admin/Common/InputField.tsx]: Not removed, but controlled separately.

**Comment by valentinyanakiev** [comment]
Screenshot_2021-01-22_12-24-12
The offset between reference fields is bigger than the rest of the form on the admin user edit page.

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Code looks OK, just have a look at the offset between references.

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**